analytics.go

```diff
@@ -387,7 +387,7 @@ func (c *client) getMarshalled(msgs []message) ([]byte, error) {
 }
 
 // Send batch request.
-func (c *client) send(msgs []message) {
+func (c *client) send(msgs []message, count int) {
```

Comment:
attempt is more appropriate?

```suggestion
func (c *client) send(msgs []message, attempt int) {
```

Reply:
Will change this.